test/test_spec.yaml

```
@@ -645,6 +613,63 @@ SciViz: # top level tab
               type: form
               tables:
                 - test_group1_simple.TableU
+    Slideshow:
+      route: /slideShow
+      grids:
+        grid1:
+          type: fixed
+          columns: 1
+          row_height: 680
+          components:
+            mydatetime:
+              x: 0
+              y: 0
+              height: 1
+              width: 1
+              type: daterangepicker
+              channel: testtest
+            myslideshow:
+              x: 0
+              y: 1
+              height: 1
+              width: 1
+              type: slideshow
+              route: /slideshowtest
+              batch_size: 3
+              chunk_size: 50
+              buffer_size: 30
+              max_FPS: 50
+              channels: [testtest]
+              restriction: >
+                def restriction(**kwargs):
+                    return dict(**kwargs)
+              dj_query: >
+                def dj_query(test_group1_simple):
+                    # SOME QUERY THAT RETURNS ONLY ONE RECORD ONE COLUMN THAT IS THE VIDEO NAME
+                    TableA, TableB = (test_group1_simple.TableA, test_group1_simple.TableB)
+                    q = ((TableA * TableB).proj(
+                        ...,
+                        #_sciviz_font='IF(a_name = "Raphael", "rgb(255, 0, 0)", NULL)',
+                        _sciviz_background='IF(a_name = "Raphael", "rgba(50, 255, 0, 0.16)", "NULL")',
+                        ))
+                    return {'query': q, 'fetch_args': {'order_by': '_sciviz_background ASC'}}
+    IFrame:
+      route: /iFrame
+      grids:
+        grid1:
+          type: fixed
+          columns: 2
+          row_height: 500
+          components:
+            IFrame Example:
+              route: /iFrameExample
+              x: 0
+              y: 0
+              height: 1
+              width: 1
+              type: iframe
+              url: https://guzman-raphael-streamlit-examp-authenticated-appsapp1app-3jm4w7.streamlit.app/?database_host=xtradb.dev.datajoint.io&context={%22try%22:%20%22this%22,%20%22for%22:%20%22once%22}&embed=true
```

We can still make this a bit cleaner. It should work with just:

```suggestion
              url: https://guzman-raphael-streamlit-examp-authenticated-appsapp1app-3jm4w7.streamlit.app/?embed=true
```

The idea is that we could have SciViz running in OIDC login mode and it would pass the `database_host` value it was configured with.

Regarding `context`, that is supposed to be an arbitrary JSON that is passed in to SciViz components (App or Page) when it is imported into another frontend project. What the `iframe` component would then do is simply pass that down into the iframe as query parameters. The idea is to provide a mechanism to provide relevant, app-specific info that SciViz could reference from it's components.